### ISSUE TITLE
New design of the country leading board 

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-web-map-client#331]
Because the previous design is not fit the plain design style of the whole app, so we redesign it, please modify the current leader board component to follow the new design.

![image](https://user-images.githubusercontent.com/5744708/144368273-a634ff1d-16b9-4cb9-b310-c6ce1bdbd8be.png)

note: please mount the component with `import { mountWithTheme as mount } from '../../models/test-utils';` to get our theme imported, check tutorial below for how to setup component sandbox.

---

Some hints for building components:

- This kind of issue is about creating a new component, it might not need to mount it in the whole app, to know how to create a component separately, please check the tutorial below.
- Design resource: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=2497%3A11584
- A tutorial of how to use the Cypress component tool to build components:
  https://github.com/Greenstand/treetracker-wallet-web#how-to-build-a-component
- We are using MaterialUI to build the app: https://material-ui.com/
- Please mount the component with `import { mountWithTheme as mount } from '../../models/test-utils';` to get our theme imported.
- Please use MaterialUI `theme.spacing(x)` as possible as we can, so the component is more responsive
- When you are ready to raise PR for the component, please attach the screenshot of the component for review.
- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

**Comment by Koda-Pig** [comment]
@dadiorchen I am working on this issue

**Comment by arcadioramos** [comment]
Can I work on this? or it's actually assigned to Koda?

**Comment by dadiorchen** [comment]
@arcadioramos yes, you can make a try, if @Koda-Pig you are still interested, I suggest you two can do a pair-programming on this, in my opinion, this is also a good way to learn from each other and good for tech communication and sharing.

**Comment by arcadioramos** [comment]
Thank you, I'll work on it tomorrow and I hope to have it ready tomorrow too

**Comment by arcadioramos** [comment]
The issue is done :) @dadiorchen
